db/migrate/20230925095916_add_indexes_to_schools.rb

```diff
@@ -0,0 +1,16 @@
+class AddIndexesToSchools < ActiveRecord::Migration[7.0]
+  def up
+    add_column :schools, :postcode_sanitised, :string
```

Comment:
This may need to be added to [the dfe analytics config](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/e6570fb748f6cc5f23a5e647e1a9caaa0b9849a3/config/analytics.yml#L223-L225)